aws/table_aws_macie2_finding.go

```diff
@@ -0,0 +1,316 @@
+package aws
+
+import (
+	"context"
+	"strings"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/macie2"
+	"github.com/aws/aws-sdk-go-v2/service/macie2/types"
+
+	macie2v1 "github.com/aws/aws-sdk-go/service/macie2"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsMacie2Finding(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_macie2_finding",
+		Description: "AWS Macie2 Finding",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.SingleColumn("id"),
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"ValidationException", "InvalidParameter"}),
+			},
+			Hydrate: getMacie2Finding,
+			Tags:    map[string]string{"service": "macie2", "action": "GetFindings"},
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listMacie2Findings,
+			Tags:    map[string]string{"service": "macie2", "action": "ListFindings"},
+			KeyColumns: []*plugin.KeyColumn{
+				{Name: "finding_type", Require: plugin.Optional, Operators: []string{"=", "<>"}},
+				{Name: "severity", Require: plugin.Optional, Operators: []string{"=", "<>"}},
+				{Name: "status", Require: plugin.Optional, Operators: []string{"=", "<>"}},
+			},
+		},
+		GetMatrixItemFunc: SupportedRegionMatrix(macie2v1.EndpointsID),
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "id",
+				Description: "The unique identifier for the finding.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "finding_arn",
```

Comment:
```suggestion
				Name:        "arn",
```

Reply:
Removed the column from the table since AWS does not treat Macie Finding as a resource type. Reference: [AWS Macie Resource Types](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/AWS_Macie.html).
Additionally, the List and Get operations do not return an ARN value in their response.